whitelist.py

```diff
@@ -0,0 +1,227 @@
+    'aa_x-funnelback-aa-occurrences-of-technique-h44','aa_x-funnelback-aa-occurrences-of-technique-h49',
+    'aa_x-funnelback-aa-occurrences-of-technique-h53','aa_x-funnelback-aa-occurrences-of-technique-h57',
+    'aa_x-funnelback-aa-occurrences-of-technique-h58','aa_x-funnelback-aa-occurrences-of-technique-h63',
+    'aa_x-funnelback-aa-occurrences-of-technique-h64','aa_x-funnelback-aa-occurrences-of-technique-h65',
+    'aa_x-funnelback-aa-occurrences-of-technique-h67','aa_x-funnelback-aa-occurrences-of-technique-h71',
+    'aa_x-funnelback-aa-occurrences-of-technique-h73','aa_x-funnelback-aa-occurrences-of-technique-h85',
+    'aa_x-funnelback-aa-occurrences-of-technique-h91','aa_x-funnelback-aa-occurrences-of-technique-h93',
+    'aa_x-funnelback-aa-occurrences-of-technique-pdf1','aa_x-funnelback-aa-occurrences-of-technique-pdf16',
+    'aa_x-funnelback-aa-occurrences-of-technique-pdf18','aa_x-funnelback-aa-occurrences-of-technique-pdf2',
+    'aa_x-funnelback-aa-occurrences-of-technique-sm1','aa_x-funnelback-aa-occurrences-of-technique-sm11',
+    'aa_x-funnelback-aa-occurrences-of-technique-sm12','aa_x-funnelback-aa-occurrences-of-technique-sm13',
+    'aa_x-funnelback-aa-occurrences-of-technique-sm14','aa_x-funnelback-aa-occurrences-of-technique-sm2',
+    'aa_x-funnelback-aa-occurrences-of-technique-sm6','aa_x-funnelback-aa-occurrences-of-technique-sm7',
+    'aa_x-funnelback-aa-occurrences-of-unique-failing-success-criterions',
+    'aa_x-funnelback-aa-occurrencesoffailingtechniques',
+    'aa_x-funnelback-aa-possibility_of_failure-occurrences-on-techniques',
+    'aa_x-funnelback-aa-passedlevel-a','aa_x-funnelback-aa-passedlevel-aa',
+    'aa_x-funnelback-aa-passedlevel-aaa','aa_x-funnelback-aa-passedlevels',
+    'aa_x-funnelback-aa-suspected_failure-occurrences-on-techniques',
+    'aa_x-funnelback-aa-set-of-failing-principles','aa_x-funnelback-aa-set-of-failing-success-criterions',
+    'aa_x-funnelback-aa-set-of-failing-techniques','aa_x-funnelback-aa-techniquesareaffectedby',
+    'aa_x-funnelback-aa-unaffected','ca_x-funnelback-raw-flesch-kincaid-readability-grade',
+    'ca_x-funnelback-rounded-flesch-kincaid-readability-grade',
+    'ca_x-funnelback-title-for-duplicate-detection','ca_x-funnelback-undesirable-text',
+)
+dta_wl = (
+    'DOCURL','HOST','PORTFOLIO','REPORT_ID','3LD','m_x-dta-agds','m_x-dta-agds-count','m_x-dta-uikit',
+    'm_x-dta-uikit-count','m_x-dta-captcha','m_x-dta-captcha-count','m_x-dta-twitter',
+    'm_x-dta-twitter-count','m_x-dta-facebook','m_x-dta-facebook-count','m_x-dta-canonical',
+    'm_x-dta-canonical-count','m_x-dta-ga-id','m_x-dta-ga-360-id','m_x-dta-reported-lang',
+    'm_x-dta-detected-lang','m_x-funnelback-twitter-hash-tags','m_x-funnelback-twitter-user-tags',
+    'm_x-dta-links-clickhere','m_x-dta-links-clickhere-value','m_x-dta-links-clickhere-count',
+    'm_x-dta-h1-count','m_x-dta-title-length','m_x-dta-title-length-size','m_x-dta-weasel-words',
```

Comment:
I'd love to replace 'weasel-words' with 'non-preferred-terms'

Reply:
This script works with those fields as already defined multiple steps in the pipleine prior.

I think this is more appropriate on a ticket about updating the data dictionary/re-filtering

Reply:
Future proofish compromise: I added 'm_x-dta-non-preferred-terms' and 'm_x-dta-non-preferred-terms-count' so once the data dictionary and subsequently the post-crawl filter scripts are updated to use those names instead it should already work.